### PR TITLE
Remove redundant DFP logo handling code

### PR DESCRIPTION
**Pull request by kelvin-chappell**
This gets rid of a lot of redundant code, and some redundant work done by the admin server.
With luck it should have no effect on the site.

/cc @guardian/commercial-dev

### CODE REVIEW

File: admin/app/dfp/DfpDataCacheJob.scala [hunk 1]
```diff
@@ -44,11 +42,7 @@ class DfpDataCacheJob(capiLookupAgent: CapiLookupAgent) extends ExecutionContext
       _ <- CustomTargetingValueAgent.refresh()
       _ <- PlacementAgent.refresh()
     } {
-      val data = loadLineItems()
-      val paidForTags = paidForTagsFinder.fromLineItems(data.lineItems)
-      capiLookupAgent.refresh(paidForTags) map {
-        _ => write(data)
-      }
+      loadLineItems()
```
Comment (JonNorman) [inline admin/app/dfp/DfpDataCacheJob.scala]: Perhaps not for this PR, but is that list exhaustive of the DFP sources we would want to refresh?

Comment (kelvin-chappell) [inline admin/app/dfp/DfpDataCacheJob.scala]: I think there are more reading jobs since this code was written, so probably not.

**Comment by regiskuckaertz** [comment]
picture 3

😍

File: facia/app/views/fragments/frontBody.scala.html [hunk 2]
```diff
@@ -1,24 +1,22 @@
...
                 "fc-container--sponsored" -> isSponsored,
-                "fc-container--advertisement-feature paid-content--advertisement-feature" -> isAdFeature,
-                "js-sponsored-front" -> (isSponsored || isAdFeature || isFoundationFunded)
+                "fc-container--advertisement-feature paid-content--advertisement-feature" -> isAdFeature
             ), "facia-page")"
             data-link-name="Front | @request.path"
-                @faciaPage.sponsor.map { sponsor =>
-                data-sponsor="@sponsor"
+                @faciaPage.branding(Edition(request)).map { branding =>
+                data-sponsor="@branding.sponsorName"
```
Comment (regiskuckaertz) [inline facia/app/views/fragments/frontBody.scala.html]: Why aren't these two data attributes removed entirely?

File: common/app/layout/Front.scala [hunk 3]
```diff
@@ -269,7 +165,7 @@ object FaciaContainer {
     // popular containers should never be sponsored
     container match {
       case MostPopular => ContainerCommercialOptions.mostPopular(omitMPU)
```
Comment (JonNorman) [inline common/app/layout/Front.scala]: Do we still need `mostPopular` and `empty` given that they each just wrap `ContainerCommercialOptions`?

File: common/app/model/meta.scala [hunk 4]
```diff
@@ -80,39 +56,14 @@ final case class Commercial(
...
-
-  def needsHighMerchandisingSlot(edition:Edition): Boolean = {
-    DfpAgent.isTargetedByHighMerch(metadata.adUnitSuffix,tags.tags,edition,metadata.url)
+  def needsHighMerchandisingSlot(edition: Edition): Boolean = {
+    DfpAgent.isTargetedByHighMerch(metadata.adUnitSuffix, tags.tags, edition, metadata.url)
   }
-
-
-  def javascriptConfig: Map[String, JsValue] = Map(
-    ("isAdvertisementFeature", JsBoolean(isAdvertisementFeature))
```
Comment (JonNorman) [inline common/app/model/meta.scala]: I assume that there isn't anything that continues to rely on this flag client-side?

**Review by JonNorman — APPROVED** [review]
A few minor questions from me and Regis but otherwise this beautiful wrenching out of DFP logos has a :+1: from me.

File: admin/test/dfp/DfpDataCacheJobTest.scala [hunk 5]
```diff
@@ -12,8 +12,6 @@ class DfpDataCacheJobTest
   with WithTestWsClient
   with WithTestContentApiClient {
```
Comment (TBonnin) [inline admin/test/dfp/DfpDataCacheJobTest.scala]: you can remove 
```
   with BeforeAndAfterAll
   with WithTestWsClient		    
   with WithTestContentApiClient {
```
if you are not using the testContentApiClient anymor

**Review by TBonnin — CHANGES_REQUESTED** [review]

File: common/app/layout/Front.scala (hunk 3 above)
Comment (kelvin-chappell) [inline common/app/layout/Front.scala]: No.  They seem pretty redundant so I'll get rid of them.

File: common/app/model/meta.scala (hunk 4 above)
Comment (kelvin-chappell) [inline common/app/model/meta.scala]: We'll soon find out! I'll do a search through the js code.

File: admin/test/dfp/DfpDataCacheJobTest.scala (hunk 5 above)
Comment (kelvin-chappell) [inline admin/test/dfp/DfpDataCacheJobTest.scala]: Thanks.  Will do.

File: common/app/model/meta.scala (hunk 4 above)
Comment (kelvin-chappell) [inline common/app/model/meta.scala]: It's already set [here](https://github.com/guardian/frontend/blob/master/common/app/model/PressedPage.scala#L37) and [here](https://github.com/guardian/frontend/blob/kc-dfp-logo-go/common/app/model/content.scala#L229) so should be safe.

**Review by TBonnin — APPROVED** [review]

**Pull request merged**